### ISSUE TITLE
win32 compatibility

### ISSUE THREAD

**Issue by imoldfella** [cmuparlay/PAM#3]
Are you interested in pull requests? I've a few patches that let me compile with visual studio, but I wasn't sure if you would be interested in supporting this as a library or if it sits as is to represent your dissertation work. Fabulous dissertation by the way! All the best with your new job.

**Comment by syhlalala** [comment]
Yes it would be great if it can work with VS! The current version can run in windows sequentially, but we haven't made it parallel so far. We appreciate it if you have patches that can make it work with visual studio :)

Also thanks for being interested in my work! Good luck to you too!

**Comment by imoldfella** [comment]
It turns out most of the patches are required to pbbslib; that is included by submodule here. I left a small pull request to see if anyone is interested, but no response so far (it's only been a few days). I'm not in a hurry, but if anyone wants this faster let me know and I could always fork it. The changes are pretty straightforward. MSVC is apparently a lot picker about saying "inline" everywhere compared to GCC. Almost everything else can be done with single MSVC specific header. OpenMP is still 2.0 on MSVC, and RIP Cilk, but Homegrown works well enough that I haven't tried to port to TBB or Concurrency Runtime yet.
